documentation/docs/examples/image360.mdx

```diff
@@ -69,14 +69,55 @@ The `transform` property from `Image360` represents the world transformation (ro
 The `Image360Visualization` object has convenience functionality related to the visualization of the 360 image such as setting the opacity of the given 360 image.
   
 ## Events
-The `Image360Collection` exposes events `image360Entered` and `image360Exited` for subscribing to enter & exit from 360 image mode.
+The `Image360Collection` exposes events `image360Entered` and `image360Exited` for subscribing to enter & exit from 360 image mode. `image360Entered` is also fired when the revision of an image is changed.
 Subscribing to the events can look like
 ```ts
 const image360Collection = await viewer.add360ImageSet('events', { site_id: 'site-S01-area-a03' });
-image360Collection.on('image360Entered', (image360: Image360) => {
+image360Collection.on('image360Entered', (image360: Image360, revision: Image360Revision) => {
     const position = new THREE.Vector3();
     position.setFromMatrixPosition(image360.transform);
     alert(`360 Image mode entered at Image position: ${JSON.stringify(position)}`);
   });
 image360Collection.on('image360Exited', () => { alert('Exited from 360 Image mode'); });
 ```
+
+## Historical images
+Some image collections include 360 images with historical revisions. You can use the `360Image` interface to manage which revision of the image is displayed. Use `getRevisions` to obtain a list of all available revisions for an image. This method returns a list of `Image360Revision` objects sorted by date, with the most recent revision at the beginning of the list.
```

Comment:
`You can use the 360Image interface to manage which revision of the image is displayed.` I guess this isn't completly true anymore? To set a revision you must go through Cognite3DViewer, right?